src/components/Person/Person.jsx

```diff
@@ -1 +1,13 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person, age, isMerried, partnerName, sex }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person}</h2>
+    {person !== 'Olya' && <p className="Person__age">I am {age}</p>}
+    {isMerried ? (
+      <p className="Person__partner">
+        {partnerName} is my {sex === 'm' ? 'wife' : 'husband'}
+      </p>
+    ) : (
+      <p className="Person__partner">I am not married</p>
```

Comment:
it's a bad practice to use nested ternary operators
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```